src/main/java/org/broadinstitute/hellbender/engine/ReferenceFileSource.java

```diff
@@ -3,10 +3,8 @@
 import htsjdk.samtools.SAMSequenceDictionary;
 import htsjdk.samtools.reference.ReferenceSequence;
 import org.broadinstitute.hellbender.exceptions.GATKException;
-import org.broadinstitute.hellbender.utils.SimpleInterval;
```

Comment:
This does not exist.  There is a file called: import org.broadinstitute.hellbender.utils.Utils;.  Is this the one I should remove?